jobs/ci-kubernetes-node-kubelet-dockerized.sh

```diff
@@ -22,8 +22,8 @@ set -o xtrace
 
 readonly testinfra="$(dirname "${0}")/.."
 
-export NODE_TEST_SCRIPT="test/e2e_node/jenkins/e2e-node-jenkins.sh"
-export NODE_TEST_PROPERTIES="test/e2e_node/jenkins/jenkins-ci.properties"
+export NODE_TEST_SCRIPT="${REPO_DIR}/test/e2e_node/jenkins/e2e-node-jenkins.sh"
```

Comment:
I think REPO_DIR refers to the path on Jenkins, and won't be valid inside the container.